{
  "uid" : "4b4757e66a1912dae1a509f688f20b0f",
  "name" : "categories",
  "children" : [ {
    "name" : "Test defects",
    "children" : [ {
      "name" : "FileNotFoundError: [Errno 2] No such file or directory: '/user/data/case_data - 副本.xls'",
      "children" : [ {
        "name" : "本地测试上传单文件接口",
        "uid" : "df1ec7058dc1b9c1",
        "parentUid" : "b0776b56c671508bc214a0e98fbf6a26",
        "status" : "broken",
        "time" : {
          "start" : 1605868886259,
          "stop" : 1605868886262,
          "duration" : 3
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ "'case_009'", "'本地测试上传单文件接口'", "''", "'{\"msg\": \"ok\"}'", "'{\"file_excel\":\"/user/data/case_data - 副本.xls\"}'", "''", "'post'", "'data'", "'upload_file/'" ]
      } ],
      "uid" : "b0776b56c671508bc214a0e98fbf6a26"
    }, {
      "name" : "OSError: [Errno 22] Invalid argument: 'D:\\\\\\x07piAutoTest\\\\data\\\\case_data - 副本.xls'",
      "children" : [ {
        "name" : "本地测试上传多文件接口",
        "uid" : "ac01432a37a72dd3",
        "parentUid" : "92eaf4ccb983cb1767803330c5788946",
        "status" : "broken",
        "time" : {
          "start" : 1605868886328,
          "stop" : 1605868886334,
          "duration" : 6
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ "'case_009'", "'本地测试上传多文件接口'", "''", "'{\"msg\": \"ok\"}'", "'{\"files\":[\"D:\\apiAutoTest\\data\\case_data - 副本.xls\", \"D:\\apiAutoTest\\data\\case_data.xlsx\"]}'", "''", "'post'", "'data'", "'upload_files/'" ]
      } ],
      "uid" : "92eaf4ccb983cb1767803330c5788946"
    } ],
    "uid" : "bdbf199525818fae7a8651db9eafe741"
  } ]
}